### ISSUE TITLE
Documentation improvement for Plugin conflict

### ISSUE THREAD

**Issue by isgroup-srl** [felipebz/zpa#95]
It should be noted that `sonar-plsql-open-plugin-2.2.0.jar` conflict with `sonar-plsql-plugin-3.3.0.2273.jar`, as the user will get the following error message:

```
Caused by: org.picocontainer.PicoCompositionException: Duplicate Keys not allowed. Duplicate for 'org.sonar.api.config.PropertyDefinition-sonar.plsql.file.suffixes'
```

**Comment by felipebz** [comment]
Thank you for reporting this. I want to change these keys, but first I'll have to check if this changes won't break everything. ;-)

**Comment by felipebz** [comment]
This was fixed in the version [2.3.0](https://github.com/felipebz/zpa/releases/tag/2.3.0). :-)

The old `sonar.plsql.*` properties are now called `sonar.zpa.*` and now this plugin can be used in a server with the SonarPLSQL plugin from SonarSource, without conflicts.

![image](https://user-images.githubusercontent.com/13829/58523979-e8ac5480-819c-11e9-9307-40fdfdf3f1c1.png)

**Comment by isgroup-srl** [comment]
Cool @felipebz ! Thanks, Francesco
